lib/github-pages-health-check/cloudflare.rb

```diff
@@ -1,9 +1,10 @@
-class GitHubPages
-  class HealthCheck
+module GitHubPages
```

Comment:
Which is the "right" way to do it? Figure it's easy to update one-off scripts. Could also be `GitHub::Pages::HealthCheck`?